packer_test/lib/base.go

```diff
@@ -1,4 +1,4 @@
-package packer_test
+package lib
```

Comment:
I have to dive into these changes further. But I hesitate to treat this as a separate package. I changed the name to packer_test, as that is the Go idiom for denoting that this is a test package, which does not have direct access to anything in the `packer` package and also to ensure that the package is excluded from the final binary. 

Previously, the go files ended with `_test. go`, which are excluded from the binary. It looks like this refactor changes that.

Reply:
I would think the code will not be part of the final binary still, as it's not transitively imported from `main.go`, that said if you think that the package name is a problem, we can still move the lib files back to the upper package so that packages using the test suite will still include something called `_test`.